functions/clash_impact.h

```diff
@@ -0,0 +1,37 @@
+#ifndef FUNCTIONS_CLASH_IMPACT_H
+#define FUNCTIONS_CLASH_IMPACT_H
+
+#include "../common/saber_base.h"
+
+// Usage: ClashImpact<MIN, MAX>
+// MIN is minimum value Clash is detected
+// MAX is maximum impact to return 32768
+// Returns 0-32768 based on impact strength of clash
+// returned value: INTEGER
+
+template<class MIN, class MAX>
+class ClashImpactF {
```

Comment:
Actually, this should be called ClashImpactFX, and ClashImpact at the bottom should be called ClashImpactF<>

Reply:
updated